HelikarLab/candis
# React data grid works now, version issue earlier.

solves #28 
###### Explanation About What Code Achieves:

  - Tested on **Chrome** web browser.
- Earlier we were using 3.0.3 version of both react-data-grid and react-data-grid-addon, which was creating error `this.setState is not defined` (possibly a binding issue), now its rectified with latest version.

###### Screenshots/GIFs:

  - GIF:

![react_data_grid_patch](https://user-images.githubusercontent.com/20624380/40533229-a56527d2-601f-11e8-8485-437e5bd79ccc.gif)
 
Its exactly similar to readme GIF ✨

###### Steps To Test:

  - empty cache if you were using earlier version, run `yarn run build` and then `python3 -m candis`.
Make sure react-data-grid is 4.0.7 and react-data-grid-addon is 4.0.8 . use `yarn info <package_name>` to see the version of above pakcages.

###### TODOs:

None
